tests/integration/conftest.py

```diff
@@ -217,7 +217,7 @@ def localhost_using_legacy_db():
     conn.commit()
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='function')
```

Comment:
Agree that this is more robust. But it will also add the overhead of a extra request for every non-session-fiddling test that uses this fixture.  It might not matter in practice, though.  If it appears to do matter, there should maybe be a separate fixture for tests that fiddle with logging in/out themselves.

Reply:
I havent personally noticed tests getting slower, but i havent used a stopwatch or anything. Are the current changes OK for this PR?

Reply:
It's good to go.   I'm just mentioning this aspect to store it also in your brain, for added redundancy :-D